app/helpers/mappings_helper.rb

```diff
@@ -0,0 +1,30 @@
+module MappingsHelper
+  def sort_mapping_list(collection)
+    hash = {}
+
+    collection.each_with_index do |mapping, i|
+      if mapping.symbol_mappings.empty?
+        set_empty_mapping!(hash, mapping, "empty#{i}")
+      else
+        add_symbol_mappings!(hash, mapping)
+      end
+    end
+
+    hash
+  end
+
+  private
+  def set_empty_mapping!(hash, mapping, name)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `name`. If it's necessary, use `_` or `_name` as an argument name to indicate that it won't be used.